doc/sphinx/language/gallina-extensions.rst

```diff
@@ -699,10 +699,10 @@ used by ``Function``. A more precise description is given below.
    Defines the not recursive function :token:`ident` as if declared with
    :cmd:`Definition`. Moreover the following are defined:
 
-    + :token:`ident`\ ``_rect``, :token:`ident`\ ``_rec`` and :token:`ident`\ ``_ind``,
+    + :n:`@ident__rect`, :n:`@ident__rec` and :n:`@ident__ind`,
```

Comment:
Restored the original; `@ident_rect` gives an undefined token error while `ident_rect` looks like `ident` is mean literally rather than something to be substituted.